OpenOversight/app/utils.py

```diff
@@ -565,36 +565,34 @@ def create_description(self, form):
 
 
 def crop_image(image, crop_data=None, department_id=None):
+    """Crops an image to given dimensions and shrinks it to fit within a configured
+    bounding box if the cropped image is still too big.
+    """
     if "http" in image.filepath:
         with urlopen(image.filepath) as response:
             image_buf = BytesIO(response.read())
     else:
         image_buf = open(os.path.abspath(current_app.root_path) + image.filepath, "rb")
 
-    image_buf.seek(0)
-    image_type = imghdr.what(image_buf)
-    if not image_type:
-        image_type = os.path.splitext(image.filepath)[1].lower()[1:]
-        if image_type in ("jp2", "j2k", "jpf", "jpx", "jpm", "mj2"):
-            image_type = "jpeg2000"
-        elif image_type in ("jpg", "jpeg", "jpe", "jif", "jfif", "jfi"):
-            image_type = "jpeg"
-        elif image_type in ("tif", "tiff"):
-            image_type = "tiff"
     pimage = Pimage.open(image_buf)
 
-    SIZE = 300, 300
+    SIZE = 1000, 1000
```

Comment:
Could we rename this to `THUMBNAIL_SIZE` and move this to the top of the file so it functions more as a module level variable?

Reply:
Done!